src/main/java/duke/Ui.java

```diff
@@ -2,15 +2,24 @@
 
 import java.util.ArrayList;
 
+/**
+ * Represents the interface that a user uses to interact with
+ * a <code>Duke</code> object.
+ */
 public class Ui {
+    /**
+     * Prints to output. Styles the input before printing.
+     * 
+     * @param inputString Input String to be printed.
+     */
     public void println(String inputString) {
         System.out.println("     --------------------\n     " + inputString + "\n     --------------------");
     }
 
     /**
-     * Uses a line-wise input array of Strings and formats it for output.
-     * @param inputStrings
-     * @return String of output.
+     * Uses a line-wise input array of Strings and styles it for output.
+     * 

```

Comment:
I like that you added one extra line separating the description and param sections as recommended :)